src/app/portlets/content-types/main/content-types.component.ts

```diff
@@ -49,72 +57,101 @@ export class ContentTypesPortletComponent extends BaseComponent {
     onMessage(): void {
         this.actionHeaderOptions = {
             primary: {
-                command: ($event) => {
+                command: $event => {
                     this.createContentType($event);
                 },
                 model: [
                     {
-                        command: ($event) => {
+                        command: $event => {
                             this.createContentType('content', $event);
                         },
                         icon: 'fa-newspaper-o',
                         label: this.i18nMessages['Content']
                     },
                     {
-                        command: ($event) => {
+                        command: $event => {
                             this.createContentType('widget', $event);
                         },
                         icon: 'fa-cog',
                         label: this.i18nMessages['Widget']
                     },
                     {
-                        command: ($event) => {
+                        command: $event => {
                             this.createContentType('file', $event);
                         },
                         icon: 'fa-file-o',
                         label: this.i18nMessages['File']
                     },
                     {
-                        command: ($event) => {
+                        command: $event => {
```

Comment:
If you do it without the curly brackets it will ```return``` its the equivalent of doing:

```
$event => {
  return this.createContentType('page', $event);
}
```

And we don't what that.